test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1

```diff
@@ -20,7 +20,7 @@ try
 
             $EICAR_STRING_B64 = "awZ8EmMWc3JjaAdvY2lrBgcbY20aBHBwGgROF3Z6cHJhHmBncn13cmF3HnJ9Z3plemFmYB5ndmBnHnV6f3YSF3sYexk= "
             $bytes = [System.Convert]::FromBase64String($EICAR_STRING_B64)
-            $EICAR_STRING = -join ($bytes | ForeEach-Object { [char]($_ -bxor 0x33) })
```

Comment:
The issue isn't the use of alias or not; the issue is that the cmdlet name was misspelled. I think removing the alias was the right move; we should keep the proper full cmdlet name.

Why did this not come up in the initial PR?

Reply:
Thanks! I'll try cmdlet name. In initial PR we did not run all tests with [Feature].

Reply:
Seems like we should be able to detect that the change is within a `Feature` test and automatically run those, let me think about this.

Reply:
CoreFX team uses a bot to run tests. We could do the same. 
Perhaps it makes sense to run short set of tests initially for speed. Then always run full set of tests before merge.

Reply:
@iSazonov that's a good idea.  Maintainers can request bot to run full set of tests after review is approved and CI tests pass.  Let me work on this.